lib/react/engine.js

```diff
@@ -0,0 +1,57 @@
+const { renderToString } = require('react-dom/server')
+const { transform } = require('./babel')
+const React = require('react')
+const fs = require('fs')
+const path = require('path')
+const dirTree = require('directory-tree')
```

Comment:
We also have a `walk-sync` module, but I don't know how different these are

Reply:
You want to try and switch to it @heiskr  so we don't have to add another package?

Reply:
It looks like directory-tree and walk-sync have different output. walk-sync creates a flat array of just the file names, where directory-tree has a nested structure. I think there might be consequences to the difference as this code seems to look at just the first level. I think it's okay to ship directory-tree for now, and we can always change it later if we find another pattern we prefer.